config/chunk-uploader.php

```diff
@@ -103,4 +103,19 @@
 
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Dropzone Options
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure the options for the Dropzone driver.
+    |
+    */
+
+    'dropzone' => [
+
+        'param' => 'file',
```

Comment:
Might be nice to make this configurable via env, so that the config file doesn't have to be published to be changed

Reply:
This value 90% of the time is the default one, in the remaining 10% I think it is better to publish and modify the config.